bundle/Resources/config/services.yaml

```diff
@@ -268,6 +268,11 @@ services:
         tags:
             - { name: data_collector, template: '@NetgenSite/data_collector/git.html.twig', id: ngsite.data_collector.git }
 
+    Netgen\Bundle\SiteBundle\ContentForms\FieldType\Mapper\BinaryFileFormMapper:
```

Comment:
None of the services use FQCN here. Can we use legacy style for this new service for consistency sake?

Reply:
Updated in https://github.com/netgen/site-bundle/pull/34/commits/f8ec9c84c32d7f15795bc4e99284889ffb17f9d0.